vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -120,9 +120,11 @@ public function run_connection_pilot() {
 			$this->update_heartbeat();
 
 			// Attempting Akismet connection given that Jetpack is connected
-			$akismet_connection_attempt = Connection_Pilot\Controls::connect_akismet();
-			if ( ! $akismet_connection_attempt ) {
-				$this->send_alert( 'Alert: Could not connect Akismet automatically.' );
+			if ( ! defined( 'VIP_AKISMET_SKIP_LOAD' ) || true !== VIP_AKISMET_SKIP_LOAD ) {
```

Comment:
I would mirror the way the conditional is written with VaultPress (variable plus a &&). I think its easier to read and we maintain consistency.

Reply:
Thank you for this! I have done as you requested in the most recent commit.